.circleci/config.yml

```diff
@@ -59,13 +60,13 @@ jobs:
           name: Build CARMA
           command: |
             source ${INIT_ENV}
-            export ROS_PARALLEL_JOBS='-j1 -l1' # Try to reduce memory consumption on build
+            export ROS_PARALLEL_JOBS='-j2 -l2' # Try to reduce memory consumption on build
             build-wrapper-linux-x86-64 --out-dir /opt/carma/bw-output bash make_with_coverage.bash -m -e /opt/carma/ -o ./coverage_reports/gcov
       - run:
           name: Run C++ Tests
           command: |
             source ${INIT_ENV}
-            export ROS_PARALLEL_JOBS='-j2 -l2' # Try to reduce memory consumption on build
+            export ROS_PARALLEL_JOBS='-j3 -l3' # Try to reduce memory consumption on build
```

Comment:
Any particular reason this is 3 but the build uses 2?
If 3 works, maybe that's preferable for both?

Reply:
Updated the config file with 3's